locale/en.json

```diff
@@ -715,5 +715,19 @@
       "extend": "Stay signed in ",
       "cancel": "Sign out"
     }
+  },
+  "additionalInfo" : {
```

Comment:
shall we rename everything to 'additionalEvidence' instead of 'additionalInfo'? I also have created this placeholder and called it 'additionalEvidence'. I think it is more accurate, but happy to leave it as it is if you prefer

Reply:
👍